content/geometry/general.adoc

```diff
@@ -85,42 +85,56 @@ The origin of the world coordinate system in ASAM OSI should also be aligned wit
 
 === Local coordinate system
 
-Top-most local coordinate frames of 3D assets (so called reference coordinate frames) in OpenMATERIAL are defined in the center of the bounding box of the corresponding asset, projected to the bottom of the bounding box.
+Top-most local coordinate frames of 3D assets (so-called reference coordinate frames) in OpenMATERIAL are defined in the center of the bounding box of the corresponding asset, projected to the bottom of the bounding box.
 By static transformations, these coordinate frames may be synchronized with reference coordinate frames in ASAM OpenSCENARIO and ASAM OSI.
 Besides reference coordinate frames, 3D assets may incorporate other local coordinate frames, see xref:../geometry/object-classes.adoc[Object classes].
 
 == Naming conventions
 
+=== General
+
 The following naming conventions apply to ASAM OpenMATERIAL geometry files:
 
-* The name of a 3D model file shall have the prefix `omg_` to indicate that the file
-complies with the ASAM OpenMATERIAL geometry specification.
+* The name of a 3D model file shall have the prefix "omg_" to indicate that the file complies with the ASAM OpenMATERIAL geometry specification.
```

Comment:
I am still a bit unsure about this. It might be unnecessary in my opinion. Lets discuss this on Thursday.

Reply:
I'd also remove the "omg_" prefix and every occurance of it in the following text. I just don't see the benefit and it could be misunderstood as "oh my god" ...